validator/test/fixtures/vm_external_ip/constraints/forbid_external_ip_blacklist/data.yaml

```diff
@@ -21,4 +21,4 @@ spec:
   parameters:
     mode: "blacklist"
     instances:
-      - //compute.googleapis.com/projects/test-project/zones/us-east1-b/instances/vm-external-ip
+      - ^//compute.googleapis.com/projects/test-project/zones/us-east1-b/instances/vm-external-ip$
```

Comment:
Do all instance names have to be regex now? I'd prefer if they didn't and we either (a) automatically found ones which are regex based or (b) specially indicated that the name is a regex. Otherwise everyone will have to worry about making their hard-coded names match a proper regex.

Reply:
I have created 2 different "match_modes", exact and regex. "Exact" is default, so existing templates do not need any new changes. 

Automatic detection of regex usage is problematic.  I can search for *, ^, $ like symbols but still there are many other valid regex symbols or patterns.